src/components/HeaderWithBackBtn/index.js

```diff
@@ -0,0 +1,3 @@
+import HeaderWithBackBtn from './HeaderWithBackBtn.vue';
+
+export default HeaderWithBackBtn;
```

Comment:
What about 
```suggestion
export * from './HeaderWithBackBtn.vue';
```

Reply:
Maybe I need double-check, but I think it don't work with Vue, I'm not sure why, in fact, is weird since Vue with composition API is not exporting anything, it's just something like:

```vue
<template>
    {{ var }}
</template>

<script setup>
   const var = 'value';
</script>
```